### DISCUSSION TITLE
Bug: Big RAM usage, no purge cache?

### DISCUSSION THREAD

**Discussion by KredensKuchenny** [eritislami/evobot#854] [Q&A]
Hi, application has a problem with big RAM usage. Every time when I'm adding new music to play RAM usage grows and grows... Is possible to purge cache after each music to reduce the RAM usage?

**Comment by eritislami** [comment]
Maybe, haven't checked this, you can use pm2 memory restart to help automate it
https://pm2.keymetrics.io/docs/usage/memory-limit/
